src/valkey-benchmark.c

```diff
@@ -2015,19 +2077,19 @@ int main(int argc, char **argv) {
         }
 
         if (test_is_selected("set")) {
-            len = valkeyFormatCommand(&cmd, "SET key%s:__rand_int__ %s", tag, data);
+            len = valkeyFormatCommand(&cmd, "SET key%s:%s %s", tag, BENCHMARK_PLACEHOLDERS[0], data);
```

Comment:
Why not keep the existing code for these commands? The explicit `__rand_int__` is easier to read IMHO.